README.md

```diff
@@ -59,10 +73,23 @@ $ npm run serve
 # open in localhost:8082
 ```
 
-## subcommands
+### subcommands
 
 * ```npm run serve``` - Opens the app in your local browser on 8082, redirects `/ipfs` and `/api` calls to your ipfs daemon on port 5001.  
 
 * ```npm run local``` - Packs up the app with vulcanize, adds it to ipfs, and opens the resulting hash in your local api gateway.  
 
 * ```npm run gateway``` - Packs the app up and runs it from the ipfs.io gateway.
+
+## Contribute
+
+Feel free to join in. All welcome. Open an [issue](https://github.com/ipfs/starlog/issues)!
+
+This repository falls under the IPFS [Code of Conduct](https://github.com/ipfs/community/blob/master/code-of-conduct.md).
+
+[![](https://cdn.rawgit.com/jbenet/contribute-ipfs-gif/master/img/contribute.gif)](https://github.com/ipfs/community/blob/master/contributing.md)
+
+## License
+
+[BSD](LICENSE.md)
```

Comment:
This should be MIT, the License found in https://github.com/ipfs/starlog/blob/master/LICENSE.md#license is only there because of a previous scaffolding tool/template that was used. However, starlog is not related with Polymer.